jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java

```diff
@@ -503,42 +503,6 @@ public void testMandatoryUNLocationCodeFieldFalseFormat() {
         .statusCode(400);
   }
 
-  // Testing with mandatory fields - Except FacilityTypeCode field
-  // Should fail as FacilityTypeCode is mandatory
-  @Test
-  public void testMandatoryFacilityTypeCodeFieldFalseFormat() {
-    Map<String, Object> map = jsonToMap(VALID_TIMESTAMP);
-
-    map.remove("facilityTypeCode");
-    given()
-        .contentType("application/json")
-        .body(map)
-        .post(TIMESTAMPS)
-        .then()
-        .assertThat()
-        .statusCode(400);
-
-    // wrong format -> ENUM
-    map.put("facilityTypeCode", "abcdfght");
-    given()
-        .contentType("application/json")
-        .body(map)
-        .post(TIMESTAMPS)
-        .then()
-        .assertThat()
-        .statusCode(400);
-
-    // Empty value
-    map.put("facilityTypeCode", ""); // Empty
-    given()
-        .contentType("application/json")
-        .body(map)
-        .post(TIMESTAMPS)
-        .then()
-        .assertThat()
-        .statusCode(400);
-  }
```

Comment:
I think we should keep this test.  The first part might need corrections to still succeed (as there are definitely timestamps where `facilityTypeCode` is mandatory).

Reply:
Good point, test case restored.